prbt_hardware_support/test/unit_tests/unittest_brake_test_utils.cpp

```diff
@@ -167,6 +167,13 @@ TEST(BrakeTestUtilsTest, testGetCurrentJointStates)
 
   auto expected_msg = joint_states_pub.getNextMessage();
 
+  // Move away from zero
+  joint_states_pub.setVelocity(0.1);
+  ros::Duration(0.1).sleep();
```

Comment:
Can we get rid of the `sleep` by using some kind of barrier instead?

Reply:
Sorry, this has to be a mistake from resolving a merge-conflict. I revert it.